tasks/metrics/flog.rake

```diff
@@ -40,3 +40,4 @@ namespace :metrics do
     end
   end
 end
+# rubocop:enable Metrics/BlockLength
```

Comment:
Why do you enable it here below all code?

Reply:
Yeah, that feels strange. Rubocop added a rule to make sure developers enable disabled rules in the same file. While this behavior is great in most use cases, it becomes strange when the whole file is affected.

I excluded *.gemspec (BlockLength) and  Rakefile (FileName) in  `config/rubocop.yml` because both are general project files. I was not sure, if you are fine with adding a wildcard rule for `tasks/**/*` (BlockLength) because only two files are affected by this. In the case you are not fine with adding a wildcard rule, I would prefer having the exclude definition in the file where it belongs, rather than listing the files explicitly in the `config/rubocop.yml`.  Would be great if Rubocop would add a `# rubocop:disable all` option.

Reply:
@zaidan Lets keep it for now to make rubocop happy.